plugins/woocommerce/src/Admin/API/MarketingCampaigns.php

```diff
@@ -168,55 +175,73 @@ public function prepare_item_for_response( $item, $request ) {
 	 * @return array Item schema data.
 	 */
 	public function get_item_schema() {
-		$schema = [
+		$schema = array(
 			'$schema'    => 'http://json-schema.org/draft-04/schema#',
 			'title'      => 'marketing_campaign',
 			'type'       => 'object',
-			'properties' => [
-				'id'         => [
+			'properties' => array(
+				'id'         => array(
 					'description' => __( 'The unique identifier for the marketing campaign.', 'woocommerce' ),
 					'type'        => 'string',
-					'context'     => [ 'view' ],
+					'context'     => array( 'view' ),
 					'readonly'    => true,
-				],
-				'channel'    => [
+				),
+				'channel'    => array(
 					'description' => __( 'The unique identifier for the marketing channel that this campaign belongs to.', 'woocommerce' ),
 					'type'        => 'string',
-					'context'     => [ 'view' ],
+					'context'     => array( 'view' ),
 					'readonly'    => true,
-				],
-				'title'      => [
+				),
+				'title'      => array(
 					'description' => __( 'Title of the marketing campaign.', 'woocommerce' ),
 					'type'        => 'string',
-					'context'     => [ 'view' ],
+					'context'     => array( 'view' ),
 					'readonly'    => true,
-				],
-				'manage_url' => [
+				),
+				'manage_url' => array(
 					'description' => __( 'URL to the campaign management page.', 'woocommerce' ),
 					'type'        => 'string',
-					'context'     => [ 'view' ],
+					'context'     => array( 'view' ),
 					'readonly'    => true,
-				],
-				'cost'       => [
+				),
+				'cost'       => array(
 					'description' => __( 'Cost of the marketing campaign.', 'woocommerce' ),
-					'context'     => [ 'view' ],
+					'context'     => array( 'view' ),
+					'readonly'    => true,
+					'type'        => 'object',
+					'properties'  => array(
+						'value'    => array(
+							'type'     => 'string',
+							'context'  => array( 'view' ),
+							'readonly' => true,
+						),
+						'currency' => array(
+							'type'     => 'string',
+							'context'  => array( 'view' ),
+							'readonly' => true,
+						),
+					),
+				),
+				'sales'      => array(
+					'description' => __( 'Sales of the marketing campaign.', 'woocommerce' ),
+					'context'     => array( 'view' ),
 					'readonly'    => true,
 					'type'        => 'object',
-					'properties'  => [
-						'value'    => [
+					'properties'  => array(
+						'value'    => array(
```

Comment:
Indentation? Not sure but looks off.

Reply:
It does align with the `currency` index.